relayer/src/services/indexerService/indexer.ts

```diff
@@ -0,0 +1,95 @@
+import { Job } from "bullmq";
+import { Connection, TransactionResponse } from "@solana/web3.js";
+import { IndexedTransaction } from "@lightprotocol/zk.js";
+import { searchBackward, searchForward } from "./search";
+
+// Unused until we move to devnet for webhooks
+async function indexStreamedTransactions({
+  connection,
+  newRawTransactions,
+  job,
+  token,
+}: {
+  connection: Connection;
+  newRawTransactions: TransactionResponse[];
+  job: Job;
+  token: any;
+}) {
+  const newParsedTransactions: IndexedTransaction[] = [];
+  //   await Promise.all(
```

Comment:
Can we remove it?

Reply:
This is a hook which is not used yet.
@SwenSchaeferjohann what do you think?

Reply:
yeah let's remove it. no big deal to add it later on

Reply:
removed